cluster/kube-network-manager.manifest

```diff
@@ -7,15 +7,24 @@
 	"containers":[{
 	    "name": "kube-network-manager",
 	    "image": "opencontrail/kube-network-manager",
-	    "args": ["--", "--public_net=10.1.4.0/24"],
+	    "command": ["/bin/sh", "-c", "/go/kube-network-manager -- --private_net=10.10.0.0/16 --portal_net=10.0.0.0/16 --public_net=10.1.0.0/16 1>>/var/log/contrail/kube-network-manager.stdout 2>>/var/log/contrail/kube-network-manager.err"],
```

Comment:
We can't hardcode this stuff...
Lets move this file to kube-network-manager.manifest.j2 and use a jinja template variable.